website2/Dockerfile

```diff
@@ -22,5 +22,6 @@ EXPOSE 3000 35729
 COPY ./docs /app/docs
 COPY ./website /app/website
 RUN yarn install
-
+RUN git config user.name "Heron Publish"
```

Comment:
What is the purpose of adding git configs inside the docker container?

Reply:
When using npm publish gh-pages it was giving errors, so I used that. Will update